aspnetcore/blazor/tutorials/movie-database-app/part-3.md

```diff
@@ -252,14 +262,10 @@ A link is rendered to navigate to the `Create` page at `/movies/create`:
 </p>
 ```
 
-The <xref:Microsoft.AspNetCore.Components.QuickGrid> component displays movie entities. `Movie` from the injected database context (`DB`) is the item provider. For each movie entity, the component displays the movie's title, release date, genre, and price. A column also holds links to edit, see details, and delete each movie entity.
-
-In the following Razor markup, notice how the context (`Context`) parameter specifies a parameter name (`movie`) for the context instance of the <xref:Microsoft.AspNetCore.Components.QuickGrid.TemplateColumn%601>. Specifying a name for the context instance makes the markup more readable (the default name for the context is simply `context`). `Movie` class properties are read from the context instance. For example, the movie identifier (`Id`) is available in `movie.Id`.
-
-The at symbol (`@`) with parentheses (`@(...)`), which is called an *explicit Razor expression*, allows the `href` of each link to include the movie entity's `Id` property in the link query string as an *interpolated string* (`$...{...}...`). For a movie identifier (`Id`) of 7, the string value provided to the `href` to edit that movie is `movies/edit?id=7`. When the link is followed, the `id` field is read from the query string by the `Edit` component to load the movie.
+The <xref:Microsoft.AspNetCore.Components.QuickGrid> component displays movie entities. The item provider is a `DbSet<Movie>` obtained from the created database context (`CreateDbContext`) of the injected database context factory (`DbFactory`). For each movie entity, the component displays the movie's title, release date, genre, and price. A column also holds links to edit, see details, and delete each movie entity.
 
 ```razor
-<QuickGrid Class="table" Items="DB.Movie">
+<QuickGrid Class="table" Items="DbFactory.CreateDbContext().Movie">
```

Comment:
Hmm...@deepchoudhery @javiercn @halter73 Don't we need to dispose the DbContext instance?  Is this potentially a bug in the scaffolded code? Every time this page renders a new DbContext will get created and it doesn't get disposed, right?